

class EventNotExistError(Exception):
    pass

class CommandExistsError(Exception):
    pass